src/jit/gentree.h

```diff
@@ -2530,6 +2530,17 @@ struct GenTreeIntConCommon : public GenTree
     {
     }
 
+    bool FitsInI8()
+    {
+        return FitsInI8(IconValue());
+    }
+
+    static bool FitsInI8(ssize_t val)
+    {
+
+        return (char)val == val;
```

Comment:
`signed char` or `int8_t` to make it more obvious?

Reply:
OK

Reply:
OK